apis/console/src/lib.rs

```diff
@@ -0,0 +1,170 @@
+#![no_std]
+
+use core::fmt;
+use core::marker::PhantomData;
+use libtock_platform as platform;
+use libtock_platform::allow_ro::AllowRo;
+use libtock_platform::share;
+use libtock_platform::subscribe::Subscribe;
+use libtock_platform::{DefaultConfig, ErrorCode, Syscalls};
+
+/// The console driver.
+///
+/// It allows libraries to pass strings to the kernel's console driver.
+///
+/// # Example
+/// ```ignore
+/// use libtock2::Console;
+///
+/// // Prints 0x45 and the app which called it.
+/// Console::print(0x45);
```

Comment:
I think this example is still not quite correct -- `Console` doesn't have a `print` method, and doing a console print doesn't automatically identify which app printed the message (that behavior is specific to LowLevelDebug).